File: 1732-find-the-highest-altitude/1732-find-the-highest-altitude.js
/**
 * @param {number[]} gain
 * @return {number}
 */
var largestAltitude = function(gain) {
    var altitude = 0;
    var maxAltitude = 0;
    var len = gain.length;
        
    for (var i = 0; i < len; i++) {
        altitude += gain[i];
        maxAltitude = Math.max(maxAltitude, altitude);
    }
        
    return maxAltitude;    
};

File: 1732-find-the-highest-altitude/1732-find-the-highest-altitude.cs
public class Solution {
    public int LargestAltitude(int[] gain) {
        int altitude = 0;
        int maxAltitude = 0;
        int len = gain.Length;
        
        for (int i = 0; i < len; i++) {
            altitude += gain[i];
            maxAltitude = Math.Max(maxAltitude, altitude);
        }
        
        return maxAltitude;                
    }
}